Source/utils/language.cpp

```diff
@@ -283,13 +283,15 @@ bool HasTranslation(const std::string &locale)
 
 bool IsSmallFontTall()
 {
-	string_view code(sgOptions.Language.szCode, 2);
+	string_view code = *sgOptions.Language.code;

```

Comment:
We should be looking only at the prefix here, not the entire code:

```suggestion
	string_view code = (*sgOptions.Language.code).substr(0, 2);
```

Reply:
Fixed